js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Sidebar.tsx

```diff
@@ -161,17 +162,32 @@ export const AssetGraphExplorerSidebar = React.memo(
       });
 
       if (groupsCount === 1) {
-        return folderNodes
-          .filter((node) => node.level === 3)
-          .map((node) => ({
-            ...node,
-            level: 1,
-          }));
+        return {
+          folderNodes: folderNodes
+            .filter((node) => node.level === 3)
+            .map((node) => ({
+              ...node,
+              level: 1,
+            })),
+          groupsCount,
+        };
       }
 
-      return folderNodes;
+      return {folderNodes, groupsCount};
     }, [graphData.nodes, openNodes]);
 
+    const {nav} = React.useContext(LayoutContext);
+
+    React.useEffect(() => {
+      if (groupsCount > 1) {
+        // Close the left sidebar if there's more than 1 group (if viewing more than 1 group then this is the global asset graph)
```

Comment:
Is it always true that 1+ group === global asset graph? If not (or even if so), is there a stricter way to be sure that the user is viewing the global asset graph?

Reply:
I supposed I can thread a prop down from the global asset graph root instead. Let me do that.